lessons/misc/open-data-intro/lesson.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: 'Opening up your data'
+visible: True
+tags:
+  - misc
+  - beginner
+
+---
+
+ - **Authors**: Derek Howard
+ - **Lesson topic**: A guided discussion on opening up your research data
+
+
+## What is open data?
+[“Open data is data that can be freely used, re-used and redistributed by anyone - subject only, at most, to the requirement to attribute and sharealike.”](http://opendatahandbook.org/guide/en/what-is-open-data/)
+
+More specifically, from the open data handbook and [opendefinition.org](http://opendefinition.org/od/2.1/en/):
+- Availability and Access: the data must be available as a whole and at no more than a reasonable reproduction cost, preferably by downloading over the internet. The data must also be available in a convenient and modifiable form.
+- Re-use and Redistribution: the data must be provided under terms that permit re-use and redistribution including the intermixing with other datasets.
+- Universal Participation: everyone must be able to use, re-use and redistribute - there should be no discrimination against fields of endeavour or against persons or groups. For example, ‘non-commercial’ restrictions that would prevent ‘commercial’ use, or restrictions of use for certain purposes (e.g. only in education), are not allowed.
+
+## [Why open data?](https://mozillascience.github.io/open-data-primers/1.1-what-and-why.html)
+- Maintain accountability
```

Comment:
A minor note. These items in the list all convert into a single paragraph. So it looks like `Maintain accountability When data is open`. I'd suggest adding a colon to the end.